phpBB/phpbb/search/backend/base.php

```diff
@@ -1,15 +1,15 @@
 <?php
 /**
-*
-* This file is part of the phpBB Forum Software package.
-*
-* @copyright (c) phpBB Limited <https://www.phpbb.com>
-* @license GNU General Public License, version 2 (GPL-2.0)
-*
-* For full copyright and license information, please see
-* the docs/CREDITS.txt file.
-*
-*/
+ *
```

Comment:
This is actually not how we have indented these block, historically at least.

Reply:
I think this is resolved.

Reply:
I will change it, I think phpstorm was who reformated this when doing copy/paste

Reply:
Currently it's resolved.
no changes required.

Reply:
If it is because of the spaces, there are a lot of files already formated like this